admiral/debug_functions.sqf

```diff
@@ -287,6 +287,61 @@ adm_debug_fnc_updateLineMarker = {
     DEBUG("admiral.debug",FMT_3("Updated line marker '%1' position between '%2' and '%3'.",_markerName,_posFrom,_posTo));
 };
 
+adm_debug_fnc_createDebugCounterMarkers = {
+    FUN_ARGS_2(_side,_xPos);
+
+    PVT_2(_marker,_markerType);
+    DECLARE(_debugMarkers) = [];
+    {
+        _markerType = _x;
+        if (_markerType == "total") then {
+            _markerType = COUNTER_DEBUG_MARKER_TOTAL; // set proper marker type for the total marker since "total" doesn't exist in cfgMarkers
+        };
+        _marker = [format ["adm_counter_%1_%2", _side,_x], [_xPos,50,0], "ICON", _markerType, [_side] call adm_debug_fnc_getSideColor, COUNTER_DEBUG_MARKER_SIZE] call adm_common_fnc_createLocalMarker;
```

Comment:
The marker creation should be extracted into a separate function something like `[_type,_side,_markerType] call adm_debug_fnc_createDebugCounterMarker`.